dbscripts/db-init.sql

```diff
@@ -208,8 +208,10 @@ CREATE TABLE ProductPayments (
 -- ProductExports
 
 CREATE TABLE ProductExports (
+	productExportUuid UUID PRIMARY KEY NOT NULL DEFAULT uuid_generate_v4(),
```

Comment:
do we still need the uuid_generate_v4 if the client is sending us the uuid? 

don't have to do it as part of this change but i think since this has recently changed, we should be making sure in all of the requests that they provide the uuid (I don't think I have been checking for this in the past)

Reply:
Yea we shouldn't. I copy pasted most of this code from the ProductTransactions page which I hacked together as fast as I can.